client/src/elm/GeoLocation/Utils.elm

```diff
@@ -2999,6 +3002,10 @@ resolveGeoSructureLabelLevel2 site =
         SiteBurundi ->
             Translate.Commune
 
+        SiteSomalia ->
+            --@todo
```

Comment:
This causes:

<img width="732" height="845" alt="Selection_1701" src="https://github.com/user-attachments/assets/5a14fcca-1a27-404d-81c4-f986df2b389f" />